files/plugins/cron_cis_audit.py

```diff
@@ -108,7 +138,7 @@ def main():
     args = parse_args(sys.argv[1:])
 
     # folder does not exist - usg-cisbenchmark likely not installed
-    if not os.path.exists(AUDIT_FOLDER):
+    if not os.path.exists(AUDIT_FOLDER) and DISTRO_VERSION < 20:
```

Comment:
Question: why don't we want to check the existence of AUDIT_FOLDER when distro > 20?

Reply:
The directory will be created on the fly by the new `usg` tooling and will only be there after first run, which causes a problem for the charm.

Reply:
Ack :ok_hand: